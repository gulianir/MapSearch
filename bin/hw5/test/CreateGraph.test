# Create a graph
CreateGraph testGraph