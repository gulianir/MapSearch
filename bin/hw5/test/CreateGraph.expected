# Create a graph
created graph testGraph